### PR TITLE
Update rating error handling

### PR DESCRIPTION
**Pull request by peeratmac**
### What's this PR do?
- Provide user with an error message when they rate a movie and the rating does not go through due to an error

### Where should the reviewer start?
- MovieShowPage file, look through handleError from action creator, how handleError and errorMessage are brought in

### How should this be manually tested?
- in updateRatings on line 16, change `event.target.value` to `100` and try to login and rate a movie, error should show on MovieShowPage

### What are the relevant tickets?
- #57 

### Screenshots
image

### CODE REVIEW

File: src/components/MovieShowPage.js [hunk 1]
```diff
@@ -12,58 +12,79 @@ export class MovieShowPage extends Component {
...
+                        ...user,
+                        ratings: ratingData.ratings
+                      };
+                      updateUser(updatedRatings);
+                    });
                   })
-                }).catch(err => console.log('.catch() error on delete'))
-              }}
-            >Reset Rating</button>
+                  .catch(err => console.log('.catch() error on delete'));
```
Comment (hfaerber) [inline src/components/MovieShowPage.js]: Could you please add the same error handling to line 76's `.catch()` for the deleteRating before we merge this PR?

File: src/components/MovieShowPage.js [hunk 2]
```diff
@@ -12,58 +12,79 @@ export class MovieShowPage extends Component {
...
+        fetchRatings(user.id).then(ratingData => {
+          const newRatings = { ...user, ratings: ratingData.ratings };
+          updateUser(newRatings);
+        });
       })
-    }).catch(err => console.log('.catch() error on update'))
+      .catch(error => {
+        this.props.handleError(
+          'Problem updating your rating, please try again later.'
+        );
```
Comment (hfaerber) [inline src/components/MovieShowPage.js]: Looks good!

**Comment by peeratmac** [comment]
@hfaerber Updated to include error handling for deleteRating as well as updateRatings.

**Comment by hfaerber** [comment]
I'm going to go ahead and merge and then I will push a PR with the updated and added tests. :)

**Pull request merged**